packages/agoric-cli/test/test-inter-cli.js

```diff
@@ -557,20 +557,11 @@ test('README: inter vbank list', async t => {
 
 test('README ex1: inter bid place by-price: printed offer is correct', async t => {
   // The README example shows "bid is broadcast" but we test only bid format.
-  const noNet = '--generate-only';
+  const noNet = '--dry-run';
   const argv =
     `node inter bid by-price --give 85IST --price 8.55 --from test-acct ${noNet}`
       .trim()
       .split(' ');
-  // TODO: txhash, height stuff
-  const expected = {
```

Comment:
I was trying to keep sort of clean expected results near the front of the test and then mangle them later if necessary. oh well.

Reply:
Gotcha, I agree the new expected value is pushing the boundaries of readability a bit. I do prefer to be more explicit in tests and minimize the amount of utilities and transforms used though.